Makefile

```diff
@@ -159,7 +159,7 @@ docker-build: ## Build docker image with the manager.
 		--tag kubeshop/kusk-gateway:$(shell git describe --tags $(shell git rev-list --tags --max-count=1)) \
 		--file ./build/manager/Dockerfile \
 		.
-	minikube image --profile kgw load kubeshop/kusk-gateway:$(shell git describe --tags $(shell git rev-list --tags --max-count=1))
+	# minikube image --profile kgw load kubeshop/kusk-gateway:$(shell git describe --tags $(shell git rev-list --tags --max-count=1)) 
```

Comment:
This might effect @kylehodgetts.

I suggest you try out `make create-env.`

Reply:
done